pkg/utils/quantity.go

```diff
@@ -0,0 +1,90 @@
+package utils
```

Comment:
Is this  a copy of another file elsewhere? Are there unit tests we can bring in too?

Reply:
It's based on the k8s resources package, just providing a parser that accepts our suffixes as well as the k8s standard ones for backwards compat. Does need some unit tests.